src/agent/command_store/CMakeLists.txt

```
@@ -9,6 +9,7 @@ include(../../cmake/CommonSettings.cmake)
 set_common_settings()
 
 add_library(CommandStore src/command_store.cpp)
+include(../../cmake/ConfigureTarget.cmake)
```

```suggestion

include(../../cmake/ConfigureTarget.cmake)
```
Please, add a space before this line.

---

Done